pkg/handlers/v1/eni.go

```diff
@@ -69,7 +82,38 @@ func (t eniTransformer) Update(event awsConfigEvent) (Output, bool, error) {
 		return Output{}, false, err
 	}
 
-	return output, filter(config), nil
+	//return output, filter(config), nil
```

Comment:
We do still want to make sure we check if the ENI is requester managed, because if it isn't then we don't really care about it. So you will want to call that filter method somewhere in this function.

Also we should probably clean this line up :)

Reply:
Oh that is true this line must've slipped by me. Thanks for the catch!